frontend/public/components/utils/details-page.tsx

```diff
@@ -200,5 +212,10 @@ export type ResourcePodCountProps = {
   resource: K8sResourceKind;
 };
 
+export type RuntimeClassProps = {
+  resource: K8sResourceKind;
```

Comment:
Prefer `K8sResourceCommon` in most places, which has stricter type checking.

```suggestion
  resource: K8sResourceCommon;
```

Reply:
Fixed,  really wish I could test these last fixes, trying to locate a cluster which has Kata Runtime/Example installed on it.